packages/jest-worker/src/child.js

```diff
@@ -68,11 +68,16 @@ function reportError(error: Error) {
     throw new Error('Child can only be used on a forked process');
   }
 
+  if (error == null) {
+    error = new Error('"null" or "undefined" thrown');
+  }
+
   process.send([
     PARENT_MESSAGE_ERROR,
-    error.constructor.name,
+    error.constructor && error.constructor.name,
     error.message,
```

Comment:
What happens when a string is thrown? `throw 'Foo'`?

Reply:
I just added a commit to allow this.